File: 2020/3회/4번.java
//다음 Java로 구현된 프로그램을 분석하여 그 실행 결과를 쓰시오. (단, 출력문의 출력 서식을 준수하시오.)
public class Test {
  public static void main(String[] args) {
    int a = 0, num = 0;
    while (a < 10) {
      a++;
      if (a % 2 == 1) {
        continue;
      }
      sum += a;
    }
    System.out.println(sum);
  }
}

//답
//30

File: 2023/2회/4번.c
//다음은 C언어의 문제이다. 알맞은 출력값을 작성하시오.
int n[3] = [73, 95, 82] 
sum = 0 
for(i=0;i<3;i++){
  sum += n[i];
}
switch(sum/30){
  case 10:
  case 9: printf(“A”);
  case 8: printf(“B”);
  case 7: 
  case 6: printf(“C”);
  default: printf(“D”);
}

//답
//BCD
